scripts/exrap_dump.sh

```diff
@@ -1232,7 +1233,7 @@ else
    def_time_window_9=3.0 # default time window for dump 9 is -3.0 to +3.0 hours
 fi
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_9} 1 lghtng
+$USHbufr_dump/bufr_dump_obs.sh $dumptime ${def_time_window_9} 1 lghtng
```

Comment:
This change is not required. The original $uscript_dump is defined in JOBSPROC_RAP_DUMP and can be retained. Please revert back to original code:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_1} 1 lghtng

Reply:
revised:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_9} 1 lghtng